bedrock/pocket/templates/pocket/firefox/new-tab-learn-more.html

```diff
@@ -73,7 +73,7 @@ <h2 class="new-tab-section-header">{{ ftl('pocket-new-tab-fuel') }}</h2>
   )%}
     <h2 class="new-tab-section-header">{{ ftl('pocket-new-tab-save-in') }}</h2>
     <p class="new-tab-section-body">{{ ftl('pocket-new-tab-built-right') }}</p>
-    <p class="new-tab-section-body">{{ ftl('pocket-new-tab-activate-your-account', firefox_learn_more='https://getpocket.com/firefox_learnmore') }}</p>
+    <p class="new-tab-section-body">{{ ftl('pocket-new-tab-activate-your-account', firefox_learn_more=url('pocket.pocket-and-firefox')) }}</p>
```

Comment:
@maureenlholland I updated the URL for this because it's already a page we migrated over to Bedrock. Should we also change its variable name to something that matches it?

Reply:
Did we change the url in bedrock? If the helper still resolves to `firefox_learnmore` I think this is fine to leave as is. If not, yes, probably best to update.

It looks clear enough in the ftl file:
# firefox_learn_more (url) - link to https://getpocket.com/firefox_learnmore

Reply:
So https://getpocket.com/firefox_learnmore actually redirects to https://getpocket.com/en/pocket-and-firefox/, which is a page we migrated over. I think I'll update the comment to show the `pocket-and-firefox` URL instead, that way we can keep the internal link as `url('pocket.pocket-and-firefox')`